README.md

```diff
@@ -52,5 +52,10 @@ For any questions, join #oshc-dev on Slack. Get an invite [here](https://opensou
 
 Note : All design related tasks have reward associated with them.
 
+<<<<<<< develop
```

Comment:
can you squash the commits? It's better not to have un-buildable code in the codebase. :wink: